config/200-config.yaml

```diff
@@ -58,3 +58,6 @@ data:
     # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
     #       for now. Use with caution.
     cluster-cert-secret: ""
+
+    # Specifies the amount of time that Kourier waits for the incoming requests.
+    stream_idle_timeout: 300s
```

Comment:
Snake case is used by Envoy but Knative usually use kebab case for the configuration. The configuration should be consistent.